tests/test_flows/test_flow.py

```diff
@@ -274,16 +283,18 @@ def test_existing_flow_exists(self):
             # check if flow exists can find it
             flow = openml.flows.get_flow(flow.flow_id)
             downloaded_flow_id = openml.flows.flow_exists(flow.name, flow.external_version)
-            self.assertEquals(downloaded_flow_id, flow.flow_id)
+            self.assertEqual(downloaded_flow_id, flow.flow_id)
 
     def test_sklearn_to_upload_to_flow(self):
         iris = sklearn.datasets.load_iris()
         X = iris.data
         y = iris.target
 
         # Test a more complicated flow
-        ohe = sklearn.preprocessing.OneHotEncoder(categorical_features=[1],
-                                                  handle_unknown='ignore')
+        ohe_params = {'handle_unknown': 'ignore'}
```

Comment:
Is there a reason why the list of categorical feature is dropped here?

Reply:
It is deprecated in scikit-learn 0.20. You should use a `ColumnTransformer` instead. The test being on numerical data (iris do not have category), I thought to just remove the parameters.

Reply:
I think the categorical feature was mostly there to have it tested in addition. Anyway, it should not really be necessary as this is also tested at some other location.